system/jhtree/jhinplace.cpp

```diff
@@ -750,12 +797,27 @@ bool PartialMatch::squash()
     return dirty;
 }
 
-const byte * PartialMatch::queryNullRow()
+const byte * PartialMatch::queryNullRow() const
 {
     return builder->queryNullRow();
 }
 
 //MORE: Pass this in...
```

Comment:
Is this comment relevant to the function you just added after it?

Reply:
I don't think it applies to either function - I will delete it